cluster-sync/sync.sh

```diff
@@ -26,9 +26,10 @@ if [ "${KUBEVIRT_PROVIDER}" != "external" ]; then
 fi
 
 # TODO: fetch it dynamically
-export OPERATOR_VERSION=0.0.1
+export OPERATOR_VERSION=v0.0.1
```

Comment:
We can update `hack/prepare-release.sh` to replace the versions to be explicit same as it does for README file - so we want have to manually manage this file.

Reply:
Right, thanks. I've just change it so we generate it for each test, so we call:
`VERSION=${OPERATOR_VERSION} TARGET_NAMESPACE=${OPERATOR_NAMESPACE} make gen-manifests`